pennylane/transforms/core/transform_dispatcher.py

```diff
@@ -56,7 +56,7 @@ def __call__(self, *targs, **tkwargs):
             # is the object we wish to transform
             obj, *targs = targs
 
-        if isinstance(obj, qml.tape.QuantumTape):
+        if isinstance(obj, (qml.tape.QuantumTape, qml.tape.QuantumScript)):
```

Comment:
```suggestion
        if isinstance(obj, qml.tape.QuantumScript):
```

Since `QuantumTape` is a `QuantumScript, we only need `QuantumScript`.